cmd/goplugin-foobernetes/main.go

```diff
@@ -0,0 +1,19 @@
+package main
+
+import (
+	"os"
+
+	"github.com/hashicorp/go-hclog"
+	"github.com/lyraproj/lyra/cmd/goplugin-foobernetes/foobernetes"
+)
+
+func main() {
+	hclog.DefaultOptions = &hclog.LoggerOptions{
+		Name:            "Puppet",
+		Level:           hclog.Debug,
+		JSONFormat:      false,
```

Comment:
I found that JSONFormat is actually great to have turned on in plug-ins. It makes the messages parse correctly in the client. I wrote about my findings in https://github.com/lyraproj/lyra/issues/110

Reply:
Nice, change pushed.